examples/client-logger/app-insights-logger/README.md

```diff
@@ -26,7 +26,7 @@ function initializeTinyliciousClient(): TinyliciousClient {
 }
 ```
 
-<!-- AUTO-GENERATED-CONTENT:START (EXAMPLE_GETTING_STARTED:usesTinylicious=FALSE) -->
+<!-- AUTO-GENERATED-CONTENT:START (EXAMPLE_README_HEADER:usesTinylicious=FALSE) -->
 
```

Comment:
Do the contribution guidelines need to be added here too?

Reply:
They're already included toward the bottom of the file

Reply:
A handful of these already contained the footer pragma, but were using only a subset of the header contents.